### ISSUE TITLE
primitive oil press

### ISSUE THREAD

**Issue by robotic-gamer** [CleverRaven/Cataclysm-DDA#55730]
### Is your feature request related to a problem? Please describe.

the lack of a way to make vegetable oil without electricity and the lack of accessibility for the parts to make an oil press for innawoods runs.

### Solution you would like.

a makeshift oil press that uses materials available for innawood runs. one possible design would be a large bulky press resembling a printing press that uses an external heat source like a fire to operate

### Describe alternatives you have considered.

replace the hotplate in the manual oil press with a lamp oil cooker or charcoal cooker and battery power for the respective fuel

### Additional context

given how long humanity has been making things like olive oil (4000 BC ish) its odd that there is no way to make vegetable oil without scavenging modern components. The easiest way to balance things if to make the primitive oil press furniture and be more time consuming then with more modern tools.

**Comment by chrispikula** [comment]
The other option for balancing is to have much worse yields.

**Comment by LeahLuong** [comment]
Were you looking for #54467??

**Comment by robotic-gamer** [comment]
that is related yes. It seems the issue with #53680 and #54467 is that they both failed to consider innawood runs where the ability to be manufactured from scratch is a must and inferiority to more modern methods and tools is acceptable. thank you for pointing out #54467 I was not aware of it.

**Comment by Light-Wave** [comment]
An oil press using an lamp oil cooker or charcoal cooker seems like a rather simple and easy addition. Both require a pilot light that requires copper, which requires mining or finding some in a cave, so that is somewhat high tech.
I also like the idea of a big and bulky printing press like structure. Perhaps one made mostly of wood?
Another quick fix would be to add the original vegetable oil recipe back to the Innawood mod.

**Comment by robotic-gamer** [comment]
when I said innawood I was referring to to runs where you build a base in the middle of nowhere with little to no scavenging in the cities or other dangerous locations not runs where you use the innawood mod. Sorry I didn't realize the way I phrased things could cause confusion.
